core-bundle/src/Resources/contao/library/Contao/FeedItem.php

```diff
@@ -101,9 +104,11 @@ public function __isset($strKey)
 	 * @param string $strUrl   The base URL
 	 * @param string $strMedia The media type
 	 */
-	public function addEnclosure($strFile, $strUrl=null, $strMedia='enclosure')
+	public function addEnclosure($strFile, $strUrl=null, $strMedia='enclosure', $size = null)
```

Comment:
`$size` is missing in the docbloc

Reply:
Fixed

Reply:
https://github.com/contao/contao/pull/3630/commits/18bac58c6522ab73dcea26867d8e851f1cdb1bda